### PR TITLE
Fixed the reference url with a clean practice

### PR DESCRIPTION
**Pull request by reeversedev**
Provided the absolute urls for the Image link so that we can use everything in a form of constants. It looks like a clean practice.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <img 
-          src={require('./images/mentor.jpg')} 
-          alt="Mentor" 
-          className="App__background-image"
-        />
+        <img src={MentorImage} alt="Mentor" className="App__background-image" />
```
[the remark this answers is not in the document]
Comment (reeversedev) [reply src/App.js]: Agreed! I'll change this in next commit.

File: src/App.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import React, { Component } from "react";
-import MentorImage from "./images/mentor.jpg";
+import mentorImage from "./images/mentor.jpg";
```
Comment (AngeloGulina) [inline src/App.js]: 👍

Comment (reeversedev) [reply src/App.js]: Surprisingly, this has not been approved yet for development branch. 
@emmawedekind don't you think it's a good practice to use constants? I believe it makes the code even more clean.

**Comment by emmawedekind** [comment]
Crap I didn't mean to resolve that conversation. Sorry yes I will merge in right now. Can you first rebase with develop so there are no merge conflicts and I'll do it? @reeversedev :)

**Comment by reeversedev** [comment]
I don't have the write access to this. Can you please give me the access?

**Comment by emmawedekind** [comment]
You just need to merge in the changes from the upstream development branch and push to your branch. You don't need write access :) @reeversedev

**Comment by WNemencha** [comment]
Useles with #6 :/

**Comment by reeversedev** [comment]
@emmawedekind I'll do this in few hours. :)

**Comment by emmawedekind** [comment]
This is no longer needed as #6 sets this as a background image. I'll close this PR.

**Pull request closed without merging**